addon/adapters/guid.js

```diff
@@ -0,0 +1,17 @@
+import JSONAPIAdapter from 'ember-data/adapters/json-api';
+
+import config from 'ember-get-config';
+import DataAdapterMixin from 'ember-simple-auth/mixins/data-adapter-mixin';
+
+export default JSONAPIAdapter.extend(DataAdapterMixin, {
+    authorizer: 'authorizer:osf-token',
```

Comment:
Any reason for using JSONAPIAdapter instead of OsfAdapter here?

Two notes to bring this in line with other models:

Auth type should be configurable for cookie or token use:
`    authorizer: config['ember-simple-auth'].authorizer,

And instead of `DataAdapterMixin`, we may want to use `GenericDataAdapterMixin` to set XHR fields needed for cookie requests. (I'd love to make that unnecessary, but eh)